Prototypes/MorganStanley.ComposeUI.HostPrototype/MorganStanley.ComposeUI.Host/MainWindow.xaml.cs

```diff
@@ -0,0 +1,78 @@
+﻿using MorganStanley.ComposeUI.Host.Mock;
+using MorganStanley.ComposeUI.Interfaces;
+using MorganStanley.ComposeUI.Prototypes.WebAppHost;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Windows;
+using WpfDemoApp;
+
+namespace MorganStanley.ComposeUI.Host;
+
+/// <summary>
+/// Interaction logic for MainWindow.xaml
+/// </summary>
+public partial class MainWindow : Window
+{
+    private readonly ICommunicationModule _communicationModule;
+    private readonly List<IApplication> _apps = new List<IApplication>();
+
+    public MainWindow()
+    {
+        _apps.Add(new HelloWorldApp());
+        _apps.Add(new WebApplication());
+        _communicationModule = new CommunicationModuleMock();
+        InitializeComponent();
+
+    }
+
+    private async Task InitializeApps()
+    {
+        var app1 = _apps[0].Initialize(_communicationModule.GetClient());
```

Comment:
Why this instead of `Task.WhenAll(_apps.Select(app => app.Initialize(_communicationModule.GetClient()))`?